### ISSUE TITLE
currkey not defined

### ISSUE THREAD

**Issue by abusedmedia** [todotoit/cryptoloji#212]
Hi @cadecairos 

I've seen you've fixed the `currkey` definition but it doesn't work on mobile yet.

After a bit of investigation it looks like uglify, in you're build system, doesn't handle properly that variable during minification process (basically it strips the definition you put and few lines below a statement try to use that variable without success, throwing the error).
This doesn't happen in my build process though, this is why it came up recently.

Long story short: I cannot fix it since it's a builder (gulp/uglify) issue, maybe it's a node_module version issue... don't know.. hope this help, though.
